lib/widgets/sendMessage.dart

```diff
@@ -77,6 +83,100 @@ class _SendMessageState extends State<SendMessage> {
+          try {
+            var picked = await FilePicker.platform.pickFiles();
+
+            if (picked != null && picked.files.first.path != null) {
+              uploadFile(widget.props, picked.files.first.path ?? '');
+            }
+          } on Exception catch (e) {
+            print('sendMessage: error in file picker web: $e');
+          }
+        },
+      );
+    } else if (Platform.isAndroid || Platform.isIOS) {
+      return PopupMenuButton<FileType>(
+        icon: Icon(Icons.attach_file),
+        onSelected: (type) async {
+          try {
+            final _paths = (await FilePicker.platform.pickFiles(
+              type: type,
+            ))
+                ?.files;
+            if (_paths != null && _paths.first.path != null) {
+              List<PapercupsAttachment> attachments =
+                  await uploadFile(widget.props, _paths.first.path ?? "");
+
+              List<String> fileIds =
+                  attachments.map((e) => e.id ?? "").toList();
+
+              if (attachments.isNotEmpty) {
+                _sendMessage(
+                  _msgFocusNode,
+                  _msgController,
+                  widget.customer,
+                  widget.props,
+                  widget.setCustomer,
+                  widget.conversation,
+                  widget.setConversation,
+                  widget.setConversationChannel,
+                  widget.conversationChannel,
+                  widget.socket,
+                  widget.setState,
+                  widget.messages,
+                  widget.sending,
+                  attachments,
+                  fileIds,
+                  true,
+                );
+              }
+            }
+          } on PlatformException catch (e) {
+            print('sendMessage: Error in file picker: $e');
+          } catch (ex) {
+            print(ex);
+          }
+        },
+        itemBuilder: (BuildContext context) => <PopupMenuEntry<FileType>>[
+          PopupMenuItem<FileType>(
+            value: FileType.any,
+            child: ListTile(
+              leading: CircleAvatar(
+                backgroundColor: widget.props.primaryColor,
+                foregroundColor: widget.textColor,
+                child: Icon(Icons.insert_drive_file_outlined),
+              ),
+              title: Text('File'),
+              contentPadding: EdgeInsets.all(0),
+            ),
+          ),
+          PopupMenuItem<FileType>(
+            value: FileType.image,
+            child: ListTile(
+              leading: CircleAvatar(
+                backgroundColor: widget.props.primaryColor,
+                foregroundColor: widget.textColor,
+                child: Icon(Icons.image_outlined),
+              ),
+              title: Text('Image'),
+              contentPadding: EdgeInsets.all(0),
+            ),
+          ),
+        ],
+      );
+    } else {
+      return IconButton(
+        icon: Icon(Icons.attachment_outlined),
+        onPressed: () {},
```

Comment:
Let's let the user know that this feature is not currently supported on their platform, or directly hide it.